app/decorators/profile_presenter.rb

```diff
@@ -2,12 +2,4 @@ class ProfilePresenter < SimpleDelegator
   def created_at
     user.created_at.to_date
   end
```

Comment:
It's probably not worth keeping `ProfilePresenter` around for this method; we can update the profile show view to use `@user.created_at.to_date` instead.